cmd/clusterctl/pkg/client/config/providers_client.go

```diff
@@ -70,24 +73,24 @@ func (p *providersClient) defaults() []Provider {
 	defaults := []Provider{
 		// cluster API core provider
 		&provider{
-			name:         ClusterAPIName,
+			name:         ClusterAPIProviderName,
 			url:          "https://github.com/kubernetes-sigs/cluster-api/releases/latest/core-components.yaml",
 			providerType: clusterctlv1.CoreProviderType,
 		},
 
 		// Infrastructure providers
 		&provider{
-			name:         "aws",
+			name:         AWSProviderName,
 			url:          "https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/latest/infrastructure-components.yaml",
 			providerType: clusterctlv1.InfrastructureProviderType,
 		},
 		&provider{
-			name:         "docker",
+			name:         DockerProviderName,
```

Comment:
We aren't maintaining github.com/kubernetes-sigs/cluster-api-provider-docker as a separate repo any more. See #1565 for example. We may want to consider removing capd from clusterctl for now, and add it back later when we have updated release artifacts for it.

Reply:
What about assuming infrastructure-components.yaml for CAPD will get released with CAPI (might be with another file name)?

Reply:
That's fair, but we haven't decided what it will look like yet.